packages/core/src/routes/connector.ts

```diff
@@ -184,10 +188,16 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
     async (ctx, next) => {
       const { phone } = ctx.guard.body;
 
-      const connector = await getConnectorInstanceByType<SmsConnectorInstance>(ConnectorType.SMS);
+      const connectorInstances = await getConnectorInstances();
+      const connector = connectorInstances.find(
+        (connector) => connector.connector.enabled && connector.metadata.type === ConnectorType.SMS
+      );
+
+      if (!connector) {
+        throw new RequestError({ code: 'connector.not_found', type: ConnectorType.SMS });
+      }
 
-      // TODO - LOG-1875: SMS & Email Template for Test
-      await connector.sendMessage(phone, 'Test', {
+      await (connector as SmsConnectorInstance).sendMessage(phone, 'Test', {
```

Comment:
maybe we can avoid using `as` and confirm type when `find`

```ts
      const connector = connectorInstances.find(
        (connector): connector is SmsConnectorInstance =>
          connector.connector.enabled && connector.metadata.type === ConnectorType.SMS
      );
```

Reply:
Fixed, PTAL.